Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -154,18 +157,24 @@ def subnext(self):
 
 
 steps['muDPGNANO_data13.0']=merge([{'-s' : 'RAW2DIGI,NANO:@MUDPG',
-                                   '--conditions':'auto:run3_data',
-                                   '-n' : '100',
-                                   '--era' : 'Run3',
-                                   '--datatier':'NANOAOD',
-                                   '--eventcontent':'NANOAOD'}])
+                                    '-n' : '100',},
+                                   steps['NANO_data13.0']])
+
```

Comment:
I imagine this, while untreated to the PR, is just that you took the chance to implement the `merge` + `steps['NANO_data13.0']` syntax also for `@MUDPG` auto NANO.

Reply:
yes, sorry indeed.

Reply:
No worries of course